test/unit/core/reducers/data/index.ts

```diff
@@ -0,0 +1,23 @@
+import * as redux from 'redux';
+import Actions from '../../../../../src/core/actions';
+import reducer from '../../../../../src/core/reducers/data';
+import * as autocomplete from '../../../../../src/core/reducers/data/autocomplete';
+import suite from '../../../_suite';
+
+suite('reducers', ({ expect, stub }) => {
+  describe('fields', () => {
+    it('should have the default state in field be an array', () => {
+      const combineReducers = stub(redux, 'combineReducers');
+
+      expect(reducer({}, { type: 'NOT AN ACTUAL ACTION',
```

Comment:
`'NOT AN ACTUAL ACTION'` should be saved as a `const` for easy re-use (and shorter lines)